### PR TITLE
Geomap: Fixed issue where KML feature attributes were not sources fro…

### PR DESCRIPTION
**Pull request by jvanulde**
…m ExtendedData elements.

### CODE REVIEW

File: src/plugins/deps/geomap-lib.js [hunk 1]
```diff
@@ -1193,7 +1193,8 @@ var componentName = "wb-geomap",
 											atts = {};
 											for ( name in layerAttributes ) {
 												if ( layerAttributes.hasOwnProperty( name ) ) {
-													atts[ layerAttributes[ name ] ] = $row.find( name ).text();
+													att = $row.find( name ).text() ? $row.find( name ).text() : $row.find( "[ name='" +  name + "']" ).text();
```
Comment (pjackson28) [inline src/plugins/deps/geomap-lib.js]: I don't think we need to cache the result in the att variable and we are doing <code>$row.find( name ).text() more than once. I suggest the following approach instead to optimize a bit better:
```js
name, rowText;
...
rowText = $row.find( name ).text();
atts[ layerAttributes[ name ] ] = rowText ? rowText : $row.find( "[ name='" +  name + "']" ).text();
```

**Comment by LaurentGoderre** [comment]
Can you fix the recommended changes and then we can merge

**Comment by pjackson28** [comment]
Please squash the commits. You have 4 different commits, one of which is a merge commit.

**Comment by LaurentGoderre** [comment]
I can merge as soon as you squash the commits

**Comment by jvanulde** [comment]
How does one squash a merge commit?

**Comment by LaurentGoderre** [comment]
```
git rebase -i HEAD~4
```

Where 4 is the number of commits to go back. Follow the instructions to squash the commits

**Comment by andre-gh** [comment]
Can this pull request be closed, has it gone stale?

**Comment by jvanulden** [comment]
This hasn't been merged yet but should be since it fixes an issue with the KML format.

**Comment by andre-gh** [comment]
I can test this, however, I need to understand the issue.  Does this reference a different issue #?

**Comment by hsrudnicki** [comment]
Is this still an issue?  Either way, please update this issue's status, whether this issue can be closed as resolved/fixed or if otherwise? Thank you

**Comment by jvanulde** [comment]
Hasn't been merged yet.

**Comment by LaurentGoderre** [comment]
Is it good to go? I defer to you @jvanulde, if it is, I will merge

**Comment by jvanulde** [comment]
It's good to go.

**Pull request merged**